demos/object_detection_demo/python/object_detection_demo.py

```diff
@@ -49,7 +49,7 @@ def build_argparser():
     args.add_argument('-at', '--architecture_type', help='Required. Specify model\' architecture type.',
                       type=str, required=True, choices=('ssd', 'yolo', 'yolov4', 'faceboxes', 'centernet', 'ctpn',
                                                         'retinaface', 'ultra_lightweight_face_detection',
-                                                        'retinaface-pytorch', 'detr'))
+                                                        'retinaface-pytorch', 'detr', 'yolox'))
```

Comment:
Please, group YOLO* models together. Also, don't forget the readme file.

Reply:
Done